src/Integration/Helpers/KnownUIContextsWrapper.cs

```diff
@@ -91,7 +91,7 @@ public class KnownUIContextsWrapper : IKnownUIContexts
 
         public IUIContext SolutionExistsAndFullyLoadedContext => new UIContextWrapper(KnownUIContexts.SolutionExistsAndFullyLoadedContext);
 
-        public IUIContext SolutionExistsAndNotBuildingAndNotDebuggingContext => new UIContextWrapper(KnownUIContexts.SolutionBuildingContext);
+        public IUIContext SolutionExistsAndNotBuildingAndNotDebuggingContext => new UIContextWrapper(KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext);
```

Comment:
Copy-and-paste error that prevented the "bind" command from being enabled.

We don't have any test for this wrapper, and it wouldn't be easy to add them.